app/(route)/(teacher)/teacher/[id]/page.tsx

```
@@ -1,3 +1,27 @@
-export default function TeacherDetail({ params }: { params: { id: string } }) {
-  return <div>teacher</div>;
+import IconTitleChip from "../../../../components/teacher/IconTitleChip";
+import ProfileInfoBox from "../../../../components/teacher/ProfileInfoBox";
+import {
+  TEACHER_STYLE_ICON,
+  TEACHER_STYLE_TEXT,
+} from "../../../../constants/teacherStyle";
+
+export default function TeacherPage({ params }: { params: { id: string } }) {
```

ESLint

[error] 'params' is defined but never used. Allowed unused args must match /^_+$/u.

(@typescript-eslint/no-unused-vars)